monai/transforms/spatial/array.py

```diff
@@ -597,6 +598,70 @@ def __call__(self, img: NdarrayOrTensor) -> NdarrayOrTensor:
         return torch.flip(img, map_spatial_axes(img.ndim, self.spatial_axis))
 
 
+class GaussianSmoothing(Transform):
```

Comment:
Hi @Can-Zhao, there is a gaussian smoothing method here https://github.com/Project-MONAI/MONAI/blob/6602303578cb7f331fce7d3bb4334396e0bfee2d/monai/transforms/intensity/array.py#L1130-L1140 (without the batch dim)

and this one (trainable): https://github.com/Project-MONAI/MONAI/blob/6602303578cb7f331fce7d3bb4334396e0bfee2d/monai/networks/layers/simplelayers.py#L442

could you please reuse any of these APIs? Let me know if they are bugs/limitations with the existing ones...

Reply:
Oh, I didn't saw them. I will reuse one of them. Thank you!!